.github/workflows/ci.yml

```diff
@@ -9,12 +9,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.9", "3.12"]
+        python-version: ["3.10", "3.13"]
```

Comment:
should we still keep 3.9... SEED is still on 3.9.

Reply:
this project sets minimum Python at 3.10 (pyproject.toml), so I changed CI to match that. Should we support 3.9? It's going to EOL this fall, no?

Reply:
I know I know... I just don't want to upgrade SEED yet. leave it as 3.10 here...